content/guides/models/track/runs/multiple-runs-per-process.md

```diff
@@ -13,25 +13,24 @@ Manage multiple runs in a single Python process. This is useful for workflows wh
 
+{{% alert title="Requirements" %}}
+To manage multiple runs in a single Python process, you must have W&B Python SDK version `v0.19.10` or newer.
+{{% /alert  %}}
 
 ## `reinit` options
 
-Use the `reinit` parameter to determine how W&B handles multiple calls to `wandb.init()`. `reinit` accepts:
+Use the `reinit` parameter to configure how W&B handles multiple calls to `wandb.init()`. The following table describes valid arguments and their effects:
 
-- `create_new`: Create a new run (`wandb.init()`) without finishing existing, active runs. 
-   * Ideal for creating and managing concurrent processes. For example, a “primary” run that remains active while you start or end “secondary” runs.
-   * W&B does not automatically switch the global `wandb.run` to new runs. You must hold onto each run object yourself. See the [example]({{< relref "multiple-runs-per-process/#example-multiple-runs-in-one-process" >}}) below for details. 
-- `finish_previous`: Finish all active runs with `run.finish()` before creating a new one run with `wandb.init()`. 
-   * Ideal when you want to break sequential sub-processes into separate individual runs.
-   * Default behaviour for non notebook environments.
-- `return_previous`: Return the most recent, unfinished run.
-   * This option does not create a new run.
-   * Default behaviour for notebook environments.
+| | Description | Creates a run? | Example use case |
+|----------------|----------------|----------------| -----------------|
+| `create_new` |Create a new run with `wandb.init()` without finishing existing, active runs. W&B does not automatically switch the global `wandb.run` to new runs. You must hold onto each run object yourself. See the [example]({{< relref "multiple-runs-per-process/#example-multiple-runs-in-one-process" >}}) below for details.  | Yes |  Ideal for creating and managing concurrent processes. For example, a “primary” run that remains active while you start or end “secondary” runs.|
+| `finish_previous` | Finish all active runs with `run.finish()` before creating a new one run with `wandb.init()`. Default behavior for non notebook environments. | Yes | Ideal when you want to break sequential sub-processes into separate individual runs. |
+| `return_previous` |  Return the most recent, unfinished run. Default behavior for notebook environments. | No | |
 
 {{% alert  %}}
-W&B does not support `create_new` mode for integrations that assume a single global run, such as Hugging Face Trainer, Keras callbacks, and PyTorch Lightning. If you use these integrations, you should run each sub-experiment in a separate process.
+W&B does not support `create_new` mode for [W&B Integrations]({{< relref "/guides/integrations/_index.md" >}}) that assume a single global run, such as Hugging Face Trainer, Keras callbacks, and PyTorch Lightning. If you use these integrations, you should run each sub-experiment in a separate process.
```

Comment:
```suggestion
W&B does not support `create_new` mode for [W&B Integrations]({{< relref "/guides/integrations/" >}}) that assume a single global run, such as Hugging Face Trainer, Keras callbacks, and PyTorch Lightning. If you use these integrations, you should run each sub-experiment in a separate process.
```
I believe that you can't link to the underbar index, only to the directory it is in. It's the one wrinkle to `relref`.